README.md

```diff
@@ -372,8 +374,6 @@ jobs:
       auth_project_number: "123456789123"
       service_account: sa-name@project-dev-123.iam.gserviceaccount.com
       image_url: ${{ needs.build.outputs.image_tag_url}} # optional, must have format <registry>/<repository>:<tag>
```

Comment:
Documentation of the input parameter `image_url` is not consistent. 

The main issue IMHO is that REAMDE.md duplicates this description. It is well documented in the source file [run-security-scans.yml]([.github](.github/workflows)/run-security-scans.yml)

Reply:
is the inconsistency whether or not you may use a digest directly or not? not sure i follow

Reply:
Yes, that's exactly what I had in mind. 
Now realizing this is part of an working example, the inline comment might be a bit misleading here. 
Perhaps it should be `# the image created by build job` or even better removed all together?

Reply:
I don't mind either way :)

Reply:
Id say its already well documented, twice actually, and less is more :)